docs/FAQ

```diff
@@ -520,7 +520,7 @@ FAQ
 
   3.3 Why doesn't my posting using -F work?
 
-  You can't simply use -F or -d at your choice. The web server that will
+  You can't simply use -F or -d as your choice. The web server that will
```

Comment:
This seems wrong. The point here is that the choice between the options is usually given by the server or what kind of operation curl should do. Maybe we should change it more to make it clearer.

How about...

"The choice between -F and -d is usually not up to you, but depends on what HTTP operation you need curl to do."